### ISSUE TITLE
WSL Screen 3 - Advanced setup

### ISSUE THREAD

**Issue by jibel** [canonical/ubuntu-desktop-installer#157]
This is the design for the stage "**Advanced setup**" of WSL OOBE

Default
![WSL Screen 3 - Advanced Setup 1](https://user-images.githubusercontent.com/1928546/125324002-bbe93600-e33f-11eb-8927-5f02a5a13edf.png)

Filled
![WSL Screen 3 - Advanced Setup 1 filled](https://user-images.githubusercontent.com/1928546/125324016-bf7cbd00-e33f-11eb-83f0-2b219c93ca5c.png)

Design specification: [Default](https://people.canonical.com/~platform/desktop/installer/WSL/#s2
), [Filled](https://people.canonical.com/~platform/desktop/installer/WSL/#s6)

**Comment by jibel** [comment]
This page must be preloaded with the default values.

**Comment by jpnurmi** [comment]
The UI is asking for the values from subiquity when entering the screen. Should we specify some defaults in the UI instead?

**Comment by jibel** [comment]
Right, it must get the values from subiquity. We'll have a look to check if GET returns really what is expected

**Comment by jpnurmi** [comment]
Actually, the default value of the mount location field will be fixed by https://github.com/canonical/ubuntu-desktop-installer/pull/345 because there's a fix to our "validated input field" included.
